apps/auth-proxy/eslint.config.js

```diff
@@ -1,9 +1,4 @@
 import baseConfig from "@acme/eslint-config/base";
 
 /** @type {import('typescript-eslint').Config} */
-export default [
-  {
-    ignores: [".nitro/**", ".output/**"],
-  },
```

Comment:
i still think these block is valuable to have as it shows people how to ignore files even if they weren't ignored by some other source